Octokit.Tests.Integration/Clients/MigrationsClientTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Octokit;
+using Octokit.Tests.Integration;
+using Xunit;
+
+public class MigrationsClientTests
+{
+    public class TheStartNewMethod
+    {
+        readonly IGitHubClient _gitHub;
+
+        public TheStartNewMethod()
+        {
+            _gitHub = Helper.GetAuthenticatedClient();
+
+        }
+
+        [IntegrationTest]
+        public async Task CanStartNewMigration()
+        {
+            var organization = Environment.GetEnvironmentVariable("OCTOKIT_GITHUBORGANIZATION");
```

Comment:
You dont need to hit the environment vars directly here... you can use `Helper.Organization` (which behind the scenes has already retrieved the env vars etc)